src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const stateArr = [stateCopy];
+
+  for (const action of actions) {
+    let currentState = { ...stateArr[stateArr.length - 1] };
```

Comment:
no need to use let, it is no primitive data

Reply:
https://imageup.ru/img50/4596128/error-js.png.html